functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,128 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the lenght of a given argument in bits. It calculates the length of the argument and then returns the result in bits.
+
+Examples:
+
+```sql
+SELECT BIT_LENGTH("TiDB");
++----------+
+| BIT_LENGTH("TiDB") |
++----------+
+| 8 (T) + 8 (i) + 8 (D) + 8 (B) = 40 bits |
++----------+
+```
+
+```sql
+SELECT CustomerName, BIT_LENGTH(CustomerName) AS BitLengthOfName FROM Customers;
+| CustomerName|BitLengthOfName |
+|-------------|----------------|
+| Albert Einstein | 120 bits |
```

Comment:
This also mixes actual output with description.
```
mysql> SELECT CustomerName, BIT_LENGTH(CustomerName) AS BitLengthOfName FROM Customers;
+--------------------+-----------------+
| CustomerName       | BitLengthOfName |
+--------------------+-----------------+
| Albert Einstein    |             120 |
| Robert Oppenheimer |             144 |
+--------------------+-----------------+
2 rows in set (0.01 sec)
```

To get something similar:
```
mysql> SELECT CustomerName, FORMAT_BYTES(BIT_LENGTH(CustomerName)*8) AS BitLengthOfName FROM Customers;
+--------------------+-----------------+
| CustomerName       | BitLengthOfName |
+--------------------+-----------------+
| Albert Einstein    | 960 bytes       |
| Robert Oppenheimer | 1.12 KiB        |
+--------------------+-----------------+
2 rows in set (0.00 sec)
```

Reply:
please clarify what you mean by "mixing actual output with description"

Reply:
The actual output should be:
+--------------------+
| BIT_LENGTH("TiDB") |
+--------------------+
|                 32 |
+--------------------+
Instead of 
+----------+
| BIT_LENGTH("TiDB") |
+----------+
| 8 (T) + 8 (i) + 8 (D) + 8 (B) = 32 bits |
+----------+
"8 (T) + 8 (i) + 8 (D) + 8 (B) = 32 bits" is explanation not actual query result.

Reply:
oh, got it. thank you @yibin87